couchdb/couchdb.rb

```diff
@@ -160,7 +160,7 @@ def monitor_couchdb(couchdb_servers, group_name)
       begin
         uri = URI.parse("#{rhost['url']}/_stats?range=60")
 
-        if rhost['user'] and rhost['password']
+        if !rhost['user'].empty? and !rhost['password'].empty?
```

Comment:
@shankarsat : Can you check if there's something incorrect in our github settings ?